float32 VoltageBattery1
float32 CurrentBattery1
float32 VoltageBattery2
float32 CurrentBattery2
